pom.xml

```diff
@@ -33,12 +40,11 @@
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
         </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter</artifactId>
-            </dependency>
-        </dependencies>
```

Comment:
you were closing out `<dependencies>` even though you had other dependencies below